### ISSUE TITLE
member-ordering breaking the code

### ISSUE THREAD

**Issue by lipis** [palantir/tslint#4775]
### Bug Report

- __TSLint version__: 5.18.0
- __TypeScript version__: 3.5.2
- __Running TSLint via__: CLI 

#### Reproduction using [TSLint Playground](https://palantir.github.io/tslint-playground/?saved=N4Igxg9gJgpiBc4A2BDAzmgBAMQhTwAOgHaaYAOATgJYBuKALjJgOYwO4QAU5KlKAW3YxKaeAWpQ0AfnFoGNYiwDaAXQDcmABYpiUJDBlyF1JWoC+ASgLmSt4iAA04CMQBm1FghBFSmQiCUAK4GaAHivmRkAUICAEYiALQQlLCKLOGYyiRRUQpBMI45uQTFJf4gKWmZAfKM1GBoiR6iDAFFfuUBKEjkOgkM1ABeMJn5o51R9rmqxfb2IOZAA)

I also created a project https://github.com/lipis/tslint-test that demonstrates the error. Just execute `yarn format` and check the `foo.ts` file.

#### TypeScript code being linted

```ts
class Foo {
  getFoo(parameters: {ids?: string[]; handles?: string[]}) {}
}

```

with `tslint.json` configuration:

```json
{
  "rules": {
    "member-ordering": [
      true,
      {
        "order": "statics-first",
        "alphabetize": true
      }
    ]
  }
}
```

#### Actual behavior

```ts
class Foo {
  getFoo(parameters: {handles?: string[]ids?: string[]; }) {}
}
```
#### Expected behavior

```ts
class Foo {
  getFoo(parameters: {handles?: string[]; ids?: string[]}) {}
}
```

**Comment by JoshuaKGoldberg** [comment]
### 💀 _**It's time!**_ 💀

**TSLint is deprecated** and no longer accepting pull requests other than security fixes. See #4534. ☠️ 
We recommend you instead use **[typescript-eslint](https://typescript-eslint.io)** to lint your TypeScript code with **ESLint**. ✅

👋 It was a pleasure open sourcing with you!

**Comment by JoshuaKGoldberg** [comment]
🤖 Beep boop! 👉  **TSLint is deprecated** 👈  and you should **switch to [typescript-eslint](https://typescript-eslint.io)**! 🤖 

🔒 This issue is being locked to prevent further unnecessary discussions. Thank you! 👋
